src/OrbitGl/Track.h

```diff
@@ -62,17 +63,16 @@ class Track : public orbit_gl::CaptureViewElement, public std::enable_shared_fro
   void OnDrag(int x, int y) override;
 
   [[nodiscard]] virtual Type GetType() const = 0;
-  [[nodiscard]] virtual bool Movable() { return !pinned_; }
```

Comment:
Why you are erasing this? This was part of the PR to not being able to move the SchedulerTrack.

Reply:
it is currently unused - was it supposed to be used in the future PRs?

Reply:
@pierricgimmig are you still planning to use it?

Reply:
The PR introduced a new method, IsFilterable that basically does what Movable does, so it's fine to remove it.